pkg/config/fixtures/testdatainner/osv-scanner-load-path.toml

```diff
@@ -0,0 +1 @@
+LoadPath = "a/b/c"
```

Comment:
note: technically due to the tags on `Config` this will get read, but we always override it so isn't not possible for this to work (which is what we want)

_however_ if someone were to refactor `tryLoadConfig` (like what I've done in #1248), then this could start getting loaded - hence, I've added this test to safeguard, with the intent to address the actual bug in a follow-up